src/main/java/io/confluent/idesidecar/restapi/resources/ConnectionsResource.java

```diff
@@ -60,10 +60,12 @@ public Uni<ConnectionsList> listConnections() {
                   .completionStage(() -> getConnectionModel(connection.getSpec().id())))
               .collect(Collectors.toList());
           if (connectionFutures.isEmpty()) {
+            Log.error("Returning no connections");
```

Comment:
I think this does not warrant the error level. What about changing the log level to debug?

Reply:
Ooops. This was a debugging statement. I'll remove.